src/plugins/advanced_settings/public/management_app/components/field/field.tsx

```diff
@@ -443,6 +444,15 @@ export class Field extends PureComponent<FieldProps> {
           />
         </EuiText>
       );
+    } else if (setting.preferBrowserSetting) {
+      return (
+        <EuiText size="xs">
+          <FormattedMessage
+            id="advancedSettings.field.browserSettingHelpText"
+            defaultMessage="This setting is overridden by user or browser preferences."
```

Comment:
@kgcreative More copy to review - this explains why these advanced settings are disabled when `theme:enableUserControl`is set to `true`.